plugins/uploadimage/plugin.js

```diff
@@ -115,7 +115,7 @@
 				// Without this isReadOnly will not works properly.
 				temp.data( 'cke-editable', 1 );
 
-				temp.appendHtml( data.dataValue );
+				temp.appendHtml( CKEDITOR.tools.htmlSafeByReview(data.dataValue, 'Content added to tempDoc to prevent XSS attacks') );
```

Comment:
Sounds good! (We're reasonably confident that this `temp` can't later be attached to the document, right?)

Reply:
SGTM